docs/setting_region.md

```diff
@@ -0,0 +1,71 @@
+# Setting GCP region
+
+## What to consider
+
+Google Cloud Platform services are available in [many
+locations](https://cloud.google.com/about/locations/) across the globe.
+You can minimize network latency and network transport costs by running your
+Dataflow job in the same region as its input bucket, output dataset, and
+temporary directory are located. More specifically, in order to run Variant
+Transforms most efficiently you should make sure all the following resources
+are located in the same region:
+* Your source bucket set by  `--input_pattern` flag.
+* Your pipeline's temporary location set by `--temp_location` flag.
+* Your output BigQuery dataset set by `--output_table` flag.
+* Your Dataflow pipeline set by `--region` flag.
+
+## Running jobs in a particular region
+Dataflow API [requires](https://beam.apache.org/blog/2019/08/22/beam-2.15.0.html)
+setting a [GCP
+region](https://cloud.google.com/compute/docs/regions-zones/#available) via
+`--region` flag to run. In addition to this requirment you might also
+choose to run Variant Transforms in a specific region following your project’s
+security and compliance requirements. For example, in order
+to restrict your processing job to Europe, update the region as follows:
+
+```bash
+# Dataflow API.
+COMMAND="/opt/gcp_variant_transforms/bin/vcf_to_bq ...
+
+docker run gcr.io/cloud-lifesciences/gcp-variant-transforms \
+  --project "${GOOGLE_CLOUD_PROJECT}" \
+  --region "${GOOGLE_CLOUD_REGION}" \
+  "${COMMAND}"
+```
+
+Note that values of `--project` and `--region` flags will be automatically
+passed as `COMMAND` args in [`piplines_runner.sh`](docker/pipelines_runner.sh).
```

Comment:
This doesn't seem relevant - it's an implementation detail?

Reply:
I agree it's implementation details, however, I'd like to contrast this method of running VT vs running it directly using the source in which case `--region` must be provided as `COMMAND` arg.

So if you don't strongly object, I'd prefer to keep this sentence.